src/tests/components/EditActivityPage.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { ACTIVITY_PLAN_ROUTE } from '../../constants/routes';
+import { EditActivityPage } from '../../components/EditActivityPage';
+import testState from '../fixtures/state';
+
+let onEditActivitySpy;
+let onRemoveActivitySpy;
+let historySpy;
+let wrapper;
+
+beforeEach(() => {
+  onEditActivitySpy = jest.fn();
```

Comment:
I suggest to create function `mountComponent` which will return a wrapper for you, it will be cleaner and you will not need this `beforeEach`. 
But it is not required change. `Up to you`